packages/bus/src/RetryStrategies/ExponentialBackoffRetryStrategy.spec.ts

```diff
@@ -0,0 +1,37 @@
+import { ExponentialBackoffRetryStrategy } from './ExponentialBackoffRetryStrategy';
+import { instance, mock, reset, verify, when } from 'ts-mockito';
+
+describe('ExponentialBackoffRetryStrategy', () => {
+  let retryStrategy!: ExponentialBackoffRetryStrategy;
+  const mockTest = mock<Test>();
+
+  beforeEach(() => {
+    retryStrategy = new ExponentialBackoffRetryStrategy({ maxDepth: 1 });
+  });
+
+  afterEach(() => {
+    reset(mockTest);
+  });
+
+  it('should not retry if function not throw error', async () => {
+    when(mockTest.test()).thenReturn();
+    await expect(
+      retryStrategy.acquire(() => instance(mockTest).test())
+    ).resolves.not.toThrow();
+    verify(mockTest.test()).once();
```

Comment:
You have to use AAA pattern. BTW the assertion is wrong (resolved promise won't throw any error).

```suggestion
    // arrange
    const input = jest.fn().mockResolvedValue(undefined);
    
    // act
    await retryStrategy.acquire(input);
    
    // assert
    expect(input).toHaveBeenCalledTimes(1);
```